packages/render-fidelity-tools/src/common.ts

```diff
@@ -231,9 +231,7 @@ export class ImageComparator {
     }
 
     let modelPixelCount = 0;
-    let whitePixelCount = 0;
-    let blackPixelCount = 0;
-    let transparentPixelCount = 0;
+    let colorlessPixelCount = 0;
```

Comment:
Just want to make sure if i understand the logic correctly. I should check whether all pixels are colorless or not, right? And a pixel is colorless if its white or black or transparent.

Reply:
Correct.